website_sale_order_shipping_modification/views/sale_portal_templates.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2025 Tecnativa - Pilar Vargas
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <template
+        id="sale_order_portal_content"
+        inherit_id="sale.sale_order_portal_content"
+    >
+        <xpath
+            expr="//div[@id='invoicing_shipping_address']/div/strong"
+            position="before"
+        >
+            <a
+                t-if="sale_order.partner_shipping_id == sale_order.partner_invoice_id and sale_order.state =='sent' and not sale_order.is_expired"
```

Comment:
Why not allowing this in `draft` state?

Reply:
Not shown in draft https://github.com/odoo/odoo/blob/16.0/addons/sale/controllers/portal.py#L37